app/views/users/registrations/school_partially_registered.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l">Your need to verify your email</h2>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>check your email for our verification link (it may be in your spam or junk folder)</li>
+      <li>or <%= govuk_link_to "register again", users_new_path %> in 24 hours</li>
```

Comment:
Out of curiosity, does the 24h part work at the moment?

Reply:
No, I have a trello ticket for it though